routes/items.js

```diff
@@ -74,17 +74,40 @@ router.put('/:id', (req, res, next) => {
 //add new item
 
 router.post('/', (req, res, next) => {
-  const {item, aisle} = req.body;
-  const newItem = {item, aisle};
+  const { item, aisle } = req.body;
+  const newItem = { item, aisle };
 
-  if(!newItem.item){
+  if (!newItem.item) {
     const err = new Error('Missing item name');
     err.status = 400;
     return next(err);
   }
 
-  items.create(newItem)
-  .then(item => {
-    if(item)
-  })
+  items
+    .create(newItem)
+    .then(item => {
+      if (item) {
+        res
+          .location(`http://${req.headers.host}/items/${item.id}`)
```

Comment:
This url would be wrong if you had attached at `/api/items`. There's a way to get express to do most of the work for you, (see [Request.baseUrl](https://expressjs.com/en/4x/api.html#req.baseUrl))